media/starboard/starboard_renderer.cc

```diff
@@ -718,10 +721,26 @@ void StarboardRenderer::OnPlayerStatus(SbPlayerState state) {
 
 void StarboardRenderer::OnPlayerError(SbPlayerError error,
                                       const std::string& message) {
-  // TODO(b/375271948): Implement and verify error reporting.
   LOG(ERROR) << "StarboardRenderer::OnPlayerError() called with code " << error
              << " and message \"" << message << "\"";
-  NOTIMPLEMENTED();
+  switch (error) {
```

Comment:
Should we have a fallback for other cases not specified now?

Reply:
For SbPlayer, we only have `kSbPlayerErrorDecode` and `kSbPlayerErrorCapabilityChanged` for errors. It should be fine here for not specifying a default case.

Reply:
Gotcha!